src/vstest.console/CommandLine/AssemblyMetadataProvider.cs

```diff
@@ -103,17 +103,31 @@ private Architecture GetArchitectureFromAssemblyMetadata(string path)
             switch (peReader.PEHeaders.CoffHeader.Machine)
             {
                 case Machine.Amd64:
-                    return Architecture.X64;
                 case Machine.IA64:
                     return Architecture.X64;
                 case Machine.Arm64:
                     return Architecture.ARM64;
                 case Machine.Arm:
                     return Architecture.ARM;
                 case Machine.I386:
-                    return Architecture.X86;
+                    // We can distinguish AnyCPU only from the set of CorFlags.Requires32Bit, but in case of Ready                                         
```

Comment:
I am surprised, it seems that the rule about un-needed ending spaces didn't trigger. Could you remove them manually?

Reply:
done thx